includes/class-wc-calypso-bridge-admin-setup-checklist.php

```diff
@@ -131,6 +135,14 @@ public function load_checklist_styles() {
 				'show' => esc_html__( 'Show completed', 'wc-calypso-bridge' ),
 			)
 		);
+		wp_localize_script(
+			'wc-calypso-bridge-setup-checklist',
+			'wccb',
+			array(
+				'ajaxUrl' => admin_url( 'admin-ajax.php' ),
+				'nonce' => wp_create_nonce( 'woocommerce_setup_active_task' ),
```

Comment:
Minor indentation here, these lines should be aligned.

Reply:
Looks like all the other lines were indented with spaces and these were the only ones using tabs.  Updated everything to tabs!